### ISSUE TITLE
Wrong emoji category is selected if the a category is removed

### ISSUE THREAD

**Issue by robinsoc** [joeattardi/picmo#173]
Hello! 

Thanks for the lib!

I experienced an issue with  the `categories` option when I build a new `EmojiButton` instance.

If I remove a category and then when I click to any category in the picker, the chosen category is not selected, even if the emojis related to the chosen category is displayed, see the screenshot below:

![Screenshot 2021-09-24 at 12-19-31 Details - alm - Tuleap](https://user-images.githubusercontent.com/18413393/134659409-03d81cba-cfb1-4fe6-9611-cb173962c72a.png)

Here the written code to instanciate the picker: 
```javascript
import { EmojiButton } from "@joeattardi/emoji-button";

const picker new EmojiButton({
        categories: [
            "smileys",
            "people",
            "food",
            "activities",
            "travel",
            "objects",
            "symbols",
            "flags",
        ],
    });
```
As you can see, the `Animals & Nature` category was removed. 
When I click on the `Food & Drinks` category, the `Activities` category is selected even if the `Food & Drink` category is displayed

Also when you click to the last category button, you get a console error. 
You can reproduce and see the error here: https://emoji-button.js.org/docs/customize under the `Specify categories` section example

:information_source: As a workaround, I use the `emojiData` option with a customized JSON which does not contain the unwanted category.

**Comment by joeattardi** [comment]
This is fixed in the upcoming 5.0.0 release. Will close the issue upon release.

**Comment by joeattardi** [comment]
This issue is resolved in `v5.0.0-beta.6`.
